### DISCUSSION TITLE
What are some good existing APIs or Online Execution Systems?

### DISCUSSION THREAD

**Discussion by LoopGlitch26** [hackclubiter/onEditor#13] [Q&A] [answered]
We can use it to integrate with an Online IDE as an initial start

**Comment by LoopGlitch26** [comment, marked as answer]
[Website](https://judge0.com/)

JudgeO is an Open Source online execution system with lots of languages in the bag.

[GitHub link](https://github.com/judge0/judge0)

Keep adding to this thread if you find something

**Comment by LoopGlitch26** [comment]
[Sphere engine](https://sphere-engine.com/compilers) is another good API. We can try its free service.

**Comment by saswatsamal** [comment]
> We can use it to integrate with an Online IDE as an initial start

Yes we will do that since we need to explore how the apis works and how the ide work in a server.
